meson.build

```diff
@@ -0,0 +1,5 @@
+project('pgroonga', 'c', version: '4.0.2')
```

Comment:
This should be updated automatically by https://github.com/pgroonga/pgroonga/blob/0b8ef3909e559dc1ae661cf656ba44c1b5d68b19/Rakefile#L117-L147 .